### PR TITLE
Design iteration and sass implementation

### PR DESCRIPTION
**Pull request by kokx**
Implemented sass and added a design iteration

### CODE REVIEW

**Comment by kokx** [comment]
Please review this quickly @jszanto or @stefanmajoor

File: module/Application/view/layout/layout.phtml [hunk 1]
```diff
@@ -111,21 +142,25 @@ function getIsActiveClass($view, array $condition)
...
+                        Postbus 513<br>
+                        5600 MB Eindhoven<br>
                         <a href="tel:+31402472815">+31 (0)40 247 2815</a><br>
                         <a href="mailto:bestuur@gewis.nl">bestuur@gewis.nl</a>
-                    </address>
+                    </address>  
+                    <a href="#"><?= $this->translate('Full details') ?></a>
                 </div>
-                <div class="col-md-6 col-sm-12">
-                    <p>&copy; 2013 - <?php echo date('Y') ?> <?= $this->translate("by WebCommissie GEWIS.") ?> <?php echo $this->translate('All rights reserved.') ?></p>
```
Comment (jszanto) [inline module/Application/view/layout/layout.phtml]: No more copyright notice?

File: module/Photo/view/photo/photo/view.phtml [hunk 2]
```diff
@@ -5,56 +5,66 @@ $this->scriptUrl()->requireUrl('member/search');
...
             <img class="img-responsive" src="<?= $this->fileUrl($photo->getLargeThumbPath()) ?>" alt=""/>
-            <?php if(!is_null($photo->getWeeklyPhoto())): ?>
-                <!-- TOOD: Make this more responsive -->
-                <div class="ribbon-container">
-                    <div class="corner-ribbon shadow">
-                        <span class="glyphicon glyphicon-certificate" aria-hidden="true"></span>
-                        <?= $photo->getWeeklyPhoto()->getWeek()->format('d-m-Y') ?>
-                    </div>
-                </div>
+        </div>
```
Comment (jszanto) [inline module/Photo/view/photo/photo/view.phtml]: It appears that some functionality has been removed here.

**Comment by jszanto** [comment]
The image file `public/data/photo32def232810850cfbd50113ffd0bec46f158728d.jpg` should not be included.

**Comment by jszanto** [comment]
Updated installation instructions in the README would be nice.

File: module/Photo/view/photo/photo/view.phtml (hunk 2 above)
Comment (kokx) [inline module/Photo/view/photo/photo/view.phtml]: Might be a problem with the rebase I had to do.

Comment (jszanto) [inline module/Photo/view/photo/photo/view.phtml]: It appears @armandduijn removed it in b914130, probably because he didn't know what its purpose was?

**Comment by kokx** [comment]
Will remove the photo (probably using filter-branch, to remove it from history entirely, since it's quite large).

For now, the installation instructions don't change. CSS development will change however, and you need some extra setup there, for which I will write instructions.

File: module/Photo/view/photo/photo/view.phtml (hunk 2 above)
Comment (kokx) [inline module/Photo/view/photo/photo/view.phtml]: Then I'm going to re-add it.

File: module/Application/view/layout/layout.phtml (hunk 1 above)
Comment (kokx) [inline module/Application/view/layout/layout.phtml]: Will also be re-added.

File: module/Application/view/layout/layout.phtml [hunk 3]
```diff
@@ -74,26 +73,58 @@ function getIsActiveClass($view, array $condition)
...
+                                                        <input type="password" name="password" placeholder="<?= $this->translate('Password') ?>" class="form-control">
+                                                    </div>
+                                                    <div class="checkbox">
+                                                        <label>
+                                                            <input type="checkbox" value="1">
+                                                            <?= $this->translate('Stay logged in') ?>
+                                                        </label>
+                                                    </div>
+                                                    <li>
+                                                        <input type="submit" value="<?= $this->translate('Login') ?>" class="btn btn-primary form-control">
```
Comment (jszanto) [inline module/Application/view/layout/layout.phtml]: This form breaks the functionality fixed by issue #193

**Comment by jszanto** [comment]
This PR also breaks all the glyph icons which were used in various places

File: module/Application/view/layout/layout.phtml (hunk 3 above)
Comment (kokx) [inline module/Application/view/layout/layout.phtml]: Can you provide an easy fix for this? Since you fixed the issue originally.

**Comment by kokx** [comment]
That is very likely due to compiling bootstrap ourselves now. Going to look into that.

File: module/Photo/view/photo/photo/view.phtml [hunk 4]
```diff
@@ -5,56 +5,66 @@ $this->scriptUrl()->requireUrl('member/search');
...
+                    </a>        
+                </li>
+                    
+                <li>
+                    <a href="<?= $this->url('photo/album', array('album_id' => $photo->getAlbum()->getId())) ?>">
+                        <?= $photo->getAlbum()->getName() ?>
+                    </a>
+                </li>
+            
+<!--
```
Comment (jszanto) [inline module/Photo/view/photo/photo/view.phtml]: I'm assuming all of this design is still very WIP? (Otherwise is is broken)

File: module/Application/view/layout/layout.phtml (hunk 3 above)
Comment (jszanto) [inline module/Application/view/layout/layout.phtml]: If possible, then rendering User\Form\Login here would be an easy fix.

File: module/Photo/view/photo/photo/view.phtml (hunk 4 above)
Comment (kokx) [inline module/Photo/view/photo/photo/view.phtml]: It seems more WIP than I thought. I think I should split the sass part into it's own PR, and rebase the design on top of that.

File: module/Application/view/layout/layout.phtml [hunk 5]
```diff
@@ -111,21 +142,25 @@ function getIsActiveClass($view, array $condition)
...
                     <img src="<?= $this->basePath() ?>/img/logo-white.png" />
                 </div>
-                <div class="col-md-4 col-sm-8">
+                <div class="col-md-4 col-sm-4">
                     <h3>Studievereniging GEWIS</h3>
                     <address>
                         MF 3.155<br>
-                        De Groene Loper 5<br>
-                        5612AZ Eindhoven<br>
+                        Postbus 513<br>
```
Comment (stefanmajoor) [inline module/Application/view/layout/layout.phtml]: Dit is nu een beetje verwarrend. Postbus 513 is het post adres, MF 3.155 is het lokaal in het bezoekadres. Dit hoort niet bij elkaar te staan. 

Ik stel voor dit gewoon hetzelfde te doen als op de oude site, en de twee dingen te scheiden. Of is er een reden voor dat alleen dit adres gegeven wordt?

File: module/Application/view/layout/layout.phtml (hunk 3 above)
Comment (jszanto) [inline module/Application/view/layout/layout.phtml]: Actually something like this probably is easier: https://gist.github.com/jszanto/90017eb8f594fc6adf93

File: module/Application/view/layout/layout.phtml [hunk 6]
```diff
@@ -74,26 +73,58 @@ function getIsActiveClass($view, array $condition)
...
+                                                            <input type="checkbox" value="1">
+                                                            <?= $this->translate('Stay logged in') ?>
+                                                        </label>
+                                                    </div>
+                                                    <li>
+                                                        <input type="submit" value="<?= $this->translate('Login') ?>" class="btn btn-primary form-control">
+                                                    </li>
+                                                    <hr/>
+                                                    <li>
+                                                        <a href="#"><?= $this->translate('Registration page') ?></a>
```
Comment (stefanmajoor) [inline module/Application/view/layout/layout.phtml]: The link to the registration page  is gone

**Comment by kokx** [comment]
Deprecating this pull request, since too much is WIP. The sass stuff is moved into PR #241
